capstone-ui/src/Components/ResultsBox/ResultsBox.jsx

```diff
@@ -59,6 +59,19 @@ export default function ResultsBox( { locationLevel, countyFIPS } ) {
 
     candidates.sort((a, b) => b.voteCount - a.voteCount);
 
+    const [selectedCandidates, setSelectedCandidates] = useState([])
+
+    function handleClick(key) {
+        setSelectedCandidates((selected) => {
+            if (selected.includes(key)) {
+                return selected.filter((candKey) => candKey != key);
+            }
+            else {
+                return selected.concat(key);
```

Comment:
Does this take into account max candidates selected as 2?

Reply:
Not yet, that's added later on. For this moment, it just allows everything to be selected.